### PR TITLE
WildFly mini conference page

### PR DESCRIPTION
**Pull request by hpehl**
This is a very first draft for the WildFly mini conference. A preview is available at https://hpehl.github.io/wildfly.org/

The PR basically includes two changes:

1. Teaser for the conference on the homepage
2. The conference page itself (made up from data as YAML, content and layout)

Things to discuss: 

- Teaser on the homepage
- The wording and structure of the conference page

Things to do, before this can go online

- Finalize the agenda
- YouTube URL of the conference
- URL for the registration form
- URL for the feedback form

If I forgot something, please let me know.

### CODE REVIEW

File: _data/conference.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  abstract:
+    - "Nostrud eu amet ullamco exercitation mollit voluptate voluptate magna et culpa tempor voluptate nulla nostrud cillum. Amet ea aute elit exercitation labore non duis exercitation in sint consectetur deserunt laboris aliquip veniam sunt nulla."
+    - "Quis consectetur sit anim in quis ex eiusmod tempor consequat do ea exercitation nisi anim. Duis dolor sint veniam deserunt dolor aliquip laboris consequat esse deserunt esse eiusmod commodo."
+    - "Dolor excepteur pariatur cillum elit ad mollit ad est esse ea cillum eiusmod cupidatat proident in esse."
+  speaker: bstansberry
+- title: WildFly Glow
+  from: "16:45"
+  to: "17:30"
+  abstract:
+    - "In this session you will be introduced with WildFly Glow, a new approach to the WildFly provisioning."
```
Comment (bstansberry) [inline _data/conference.yaml:33]: s/introduced with/introduced to/g

**Comment by bstansberry** [comment]
We should include a simple blog post, as people may follow news as their way to track interesting things.

**Comment by hpehl** [comment]
The PR for the news post about the conference is here: https://github.com/wildfly/wildfly.org/pull/532

File: _layouts/conference.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        Virtually on YouTube Live: <a href="{{site.baseurl}}/conference" target="_blank">https://link-to-you-tube</a>
+      </dd>
+    </dl>
+    <p style="margin-bottom: 0">Registration is not required, but you can let us know that you <a href="{{site.baseurl}}/conference" target="_blank">plan to attend</a>.</p>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-yellow">
+    <h2>What you can expect</h2>
+    <p>The conference includes four sessions with topics for both beginners and advanced WildFly users. There will be an opening and closing ceremony and a 15-minute break between the second and third session. We want all sessions to be interactive, and you can ask questions at any time. Remember this is a conference for you - the WildFly community!</p>
+    <p>All times are in UTC.</p>
```
Comment (jamezp) [inline _layouts/conference.html:29]: This is just more of a thought. Should we make this bold? Another option would be to add `(UTC)` after each time.

**Review by jamezp — COMMENTED** [review]
Just one minor comment, but the layout and current content look good to me.

File: _layouts/conference.html (hunk 2 above)
Comment (fjuma) [reply _layouts/conference.html:29]: +1 for adding UTC after each time as well.

File: _layouts/conference.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+<div class="about-page grid-wrapper conference">
+  <div class="grid__item width-12-12">
+    <h1>Wild<strong>Fly</strong> Mini Conference</h1>
+    <h2 class="page-subtitle">A dedicated conference for the community with topics around Wild<strong>Fly</strong>.</h2>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-red">
+    <h2>What you need to kow</h2>
```
Comment (fjuma) [inline _layouts/conference.html:12]: s/kow/know

File: _layouts/conference.html (hunk 2 above)
Comment (jamezp) [reply _layouts/conference.html:29]: This is just a random idea, but what about a popover that shows the date in the users current time? A quick example would look something like this https://github.com/hpehl/wildfly.org/compare/conference...jamezp:wildfly.org:date-conversion. Note the position of the popover is kind of odd so we'd want to sort that out, but just an idea :)

Comment (hpehl) [reply _layouts/conference.html]: Adding "UTC" to every time breaks the layout - especially on mobile devices. Instead I use @jamezp suggestion and show the local time when clicking on the time.

File: _layouts/conference.html (hunk 3 above)
Comment (hpehl) [reply _layouts/conference.html]: Fixed

File: _layouts/conference.html (hunk 2 above)
Comment (hpehl) [reply _layouts/conference.html]: I've updated the preview at https://hpehl.github.io/wildfly.org/
WDYT?

Comment (fjuma) [reply _layouts/conference.html]: That's pretty cool! For some reason, when I click on it though, I'm still seeing UTC time, e.g., for the Welcome session, I'm seeing 2:00-2:15pm.

Comment (hpehl) [reply _layouts/conference.html]: There was an issue in the JS code. Should be fixed now. @fjuma can you verify?

Comment (fjuma) [reply _layouts/conference.html]: Perfect, that works for me now. Thanks @hpehl!

**Comment by hpehl** [comment]
Slightly changed the layout of the conference page and added the speaker's bio: https://hpehl.github.io/wildfly.org/

**Review by jamezp — APPROVED** [review]
LGTM

File: _layouts/conference.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      {% endif %}
+    </div>
+    {% endfor %}
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-teal">
+    <h2>After the conference</h2>
+    <p>Please take some time to answer a few questions about the conference. What did you like, what can we do
+      better? It won't take long, and will help us make the next WildFly conference even better: <a
+              href="{{site.baseurl}}/conference" target="_blank">conference feedback</a></p>
```
Comment (PrarthonaPaul) [inline _layouts/conference.html:79]: I think on the preview right now the link for conference feedback opens the conference page in a new tab as opposed to the feedback form.

File: _layouts/conference.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-red">
+    <h2>What you need to know</h2>
+    <dl class="c-when-and-where">
+      <dt>Date</dt>
+      <dd id="c-date" data-date="2024-03-06">Wednesday, March 6, 2024</dd>
+      <dt>Time</dt>
+      <dd>14:00 - 18:00 UTC</dd>
+      <dt>Location</dt>
```
[the remark this answers is not in the document]
Comment (hpehl) [reply _layouts/conference.html:19]: Hmm, it seems to be ok on my side:
CleanShot 2024-02-06 at 16 37 46@2x

[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply _layouts/conference.html:19]: Right, it is aligned on the right side. I did not notice that and thought they were aligned left.

**Comment by hpehl** [comment]
I've pushed some final changes (see commit message). I think we're almost done. Let's discuss in tomorrow's call when to take this live.

**Comment by hpehl** [comment]
I've added and linked an ICS file to make adding the event to your calendar easy. @bstansberry I've put the ICS file in `assets/data` for lack of a better place. Is that ok for you?

**Comment by hpehl** [comment]
Everything's ready, IMO. Removing the `hold` label.

File: _layouts/conference.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+      <dt>Date</dt>
+      <dd id="c-date" data-date="2024-03-06"><a href="{{site.baseurl}}/assets/data/wildfly-mini-conference.ics" title="Click here to download an ICS file for the conference" type="text/calendar" download>Wednesday, March 6, 2024</a></dd>
+      <dt>Time</dt>
+      <dd>14:00 - 18:00 UTC</dd>
+      <dt>Location</dt>
+      <dd>
+        Virtually on <a href="https://www.youtube.com/live/_8g1rZ80u2A" target="_blank">YouTube Live</a>
+      </dd>
+    </dl>
+    <p>Registration is not required, but you can let us know that you <a href="https://forms.gle/zK6apuNk6rotXrkq5" target="_blank">plan to attend</a>. Click on the date to add the event to your calendar. You can also use the "Notify Me" button on YouTube if you would like to be notified about the conference.</p>
```
Comment (fjuma) [inline _layouts/conference.html:24]: Just a very minor comment, might be good to move "You can also use the "Notify Me"..." button to a new line just to make it stand out a bit more.

Comment (hpehl) [reply _layouts/conference.html]: Like this: https://hpehl.github.io/wildfly.org/conference/

Comment (fjuma) [reply _layouts/conference.html]: Looks good, thanks!

**Pull request merged**